jfauthprovider/views.py

```diff
@@ -77,6 +77,7 @@ def register(request):
         "grant_type": "password",
         "username": request.POST.get("username"),
         "password": request.POST.get("password"),
+        "user_type": request.POST.get("user_type"),
```

Comment:
referring back to the first comment, the python side of this can be 
`"user_type": request.POST.get("userType")`

Reply:
Thanks for catching this. After looking back in AuthClient I realized all the other methods made the change there so I followed that for consistency